apps/tailwind-components/components/form/Legend.vue

```diff
@@ -14,7 +14,8 @@
         <span
           class="pl-4 text-title capitalize"
           :class="{ 'font-bold': section.isActive }"
-          >{{ section.label }}</span
+        >
+          {{ section.title }}</span
         >
         <span v-if="(section.errorCount ?? 0) > 0" class="ml-2">
```

Comment:
Like some of the other labels, the link between the form section title and the number of errors is missing. It might be the case that error count is wrapped in the section title and context is given.

```vue
<a ...>
  {{ section.title }}
  <template v-if="...">
    <span class="sr-only">errors</span>
    <span ...>{{ section.errorCount }}</span>
  </template>
</a>
```